bit_manipulation/count_of_set_bits.cpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file
+ * @brief Program to Count set bits in an integer
+ *
+ * @details
```

Comment:
```suggestion
 * @details
 * We are given an integer number. Let’s say, number. The task is to first calculate the
 * binary digit of a number and then calculate the total set bits of a number.
 *
 * Set bits in a binary number is represented by 1. Whenever we calculate the binary number of an integer value
 * it is formed as the combination of 0’s and 1’s. So digit 1 is known as a set bit in computer terms.
```

Reply:
If you're not going to add this, please add a detailed description of the algorithm. I'm not trying to be rude, by the way. :)